modules/customize-certificates-add-service-serving.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="add-service-certificate-{context}"]
+= Add a service certificate
+
+To secure communication to your service, have the cluster generate a 
+signed serving certificate/key pair into a secret in your namespace. To 
+do this, set the `service.alpha.openshift.io/serving-cert-secret-name` 
```

Comment:
service.beta now

Reply:
this goes for the rest of the service annotations we mention in these docs

Reply:
All of these references have been updated to `service.beta.openshift...`.